julia.html.markdown

```diff
@@ -832,9 +832,9 @@ code_native(circle_area, (Int32,), syntax = :intel)
     #         nop     dword ptr [rax]
     # ;}
 
-code_native(circle_area, (Float64,), syntax = :intel)
+codeNative(circleArea, (Float64,), syntax = :intel)
     #         .text
-    # ; Function circle_area {
+    # ; Function circleArea {
```

Comment:
Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194